Fix CSS of extension-requests page

Aryex82 opened the pull request:
## Issue: #377 

### Description:
Changed the CSS to fix the cards displayed extension requests page.

### Anthing you would like to inform the reviewer about:

### Dev Tested:
- [x] Yes


### Images/video of the change:

before: 

<img width="1352" alt="image" src="https://github.com/Real-Dev-Squad/website-dashboard/assets/98796547/a65d6015-94c7-4c60-b103-ddfed67b6ca7">

after: 


https://github.com/Real-Dev-Squad/website-dashboard/assets/98796547/347df4b8-eb85-460f-a459-e161bce60828





## Follow-up Issues (if any)

sahsisunny commented:
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/70854507/5469921b-164e-4295-b320-50135cae4acf)
Can you style the button in one ROW with equal width?

sahsisunny commented:
And also share videos that show responsiveness.

sahsisunny commented on extension-requests/style.css:
Please use REM instead for PX if the value is greater than 8px.

Aryex82 commented:
> And also share videos that show responsiveness.

added

Aryex82 commented:
> ![image](https://user-images.githubusercontent.com/70854507/246337823-5469921b-164e-4295-b320-50135cae4acf.png)
> Can you style the button in one ROW with equal width?

done

Pratiyushkumar commented on extension-requests/style.css:
```diff
@@ -40,7 +40,7 @@ body {
 .container {
   width: 100%;
   text-align: center;
-  padding: 10px;
+  padding: 40px;
 }
```
please use rem instead of px, use px only where the size is fixed for all screen-sizes.

Aryex82 replied:
done

Pratiyushkumar commented on extension-requests/style.css:
```diff
@@ -187,42 +192,53 @@ body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: auto;
   text-align: left;
   border: 2px solid var(--black-transparent);
   padding: 1.25rem;
```
also try using even measurements which helps in proper debugging and makes code look good

Aryex82 replied:
Im not sure what you are referring to here. Can you please elaborate more.

Pratiyushkumar reviewed (changes requested):
as commented by Sunny https://github.com/Real-Dev-Squad/website-dashboard/pull/378#issuecomment-1594187682
it says both the button should be in one row and with equal width and when I see the SS  the issue looks similar as before also share a clear video of the feature because video is blur

Aryex82 commented:
> as commented by Sunny [#378 (comment)](https://github.com/Real-Dev-Squad/website-dashboard/pull/378#issuecomment-1594187682)
> it says both the button should be in one row and with equal width and when I see the SS the issue looks similar as before also share a clear video of the feature because video is blur

Updated the video. Buttons when the were of equal length looked a bit odd to me, anyways I've changed it to fixed length now.

sahsisunny reviewed (approved):
LGTM🎉🎉

Aryex82 commented:
> as commented by Sunny [#378 (comment)](https://github.com/Real-Dev-Squad/website-dashboard/pull/378#issuecomment-1594187682)
> it says both the button should be in one row and with equal width and when I see the SS the issue looks similar as before also share a clear video of the feature because video is blur

done

Pratiyushkumar reviewed (approved):
LGTM
